shaka-lab-gateway/linux/config-templates/db.lab.shaka.template

```diff
@@ -0,0 +1,45 @@
+{#
```

Comment:
GitHub doesn't recognize this, but this file is renamed and slightly modified from `db.lab.template`.  All instances of `.lab` have been replaced with `.lab.shaka`.  I also added the section with SRV records, to replace the DNS records Samba was injecting in the old setup.